packages/docs-reanimated/docs/advanced/dispatchCommand.mdx

```diff
@@ -4,71 +4,78 @@ sidebar_position: 7
 
 # dispatchCommand
 
-import DocsCompatibilityInfo from '../_shared/_docs_compatibility_info.mdx';
+import DispatchCommandSrc from '!!raw-loader!@site/src/examples/DispatchCommand';
 
-<DocsCompatibilityInfo />
+`dispatchCommand` allows you to run commands on a native component. It happens on UI thread.
 
-Allows to dispatch command on a native component synchronously from the UI thread.
+## Reference
+
+```javascript
+import { dispatchCommand } from 'react-native-reanimated';
+
+function App() {
+  const animatedRef = useAnimatedRef()
+
+  const gesture = Gesture.Tap().onStart(() => {
+    dispatchCommand(animatedRef, 'focus)
+  })
```

Comment:
This snippet lacks some semicolons, one quotation mark and would be great to add a highlight
```suggestion
  const animatedRef = useAnimatedRef();

  const gesture = Gesture.Tap().onStart(() => {
    // highlight-next-line
    dispatchCommand(animatedRef, 'focus');
  })
```